addon/controllers/events.js

```diff
@@ -0,0 +1,9 @@
+import Controller from '@ember/controller';
+import { getOwner } from '@ember/application';
+
+export default class EventsController extends Controller {
+  get showBackLink() {
+    const appConfig = getOwner(this).resolveRegistration('config:environment');
```

Comment:
I *think* the way to do this has changed to [getConfig](https://github.com/embroider-build/embroider/tree/main/packages/macros#getownconfig-getconfig-and-getglobalconfig), but I haven't messed with it a lot. I'm ok with this solution, it works for sure, but we're already importing and using these macros other places so it's worth investigating.